### DISCUSSION TITLE
Clicking into inline styles on a html element triggering unpredictable cursor behaviour

### DISCUSSION THREAD

**Discussion by NickSheehy** [phcode-dev/phoenix#1980] [Q&A] [answered]
When clicking into the inline styles on a html element, the inline style code highlights as active. Once you start clicking between properties and values, the cursor/code jumps around in an unpredictable way. I'm not sure if this is part of an update, but I've not noticed the behaviour before. Is this something I can turn off? It's frustrating.

Video attached

https://github.com/user-attachments/assets/4527ce57-a812-43a5-961e-1a2179452280

**Comment by abose** [comment, marked as answer]
Apologies, this bug happens in dark theme and was a break introduced in this release. It has been fixed for the next release by https://github.com/phcode-dev/phoenix/pull/1971

Till the next release, the fix is to use the light theme or any theme from the extension store.

**Reply by NickSheehy** [reply]
Great! thanks
